Felienne/hedy
# Fix: Pop-up to "save program" shown when not signed in

Provide a general summary of your changes in the Title of the PR

**Description**

In this PR we fix the issue where the pop-up to save a changed program is shown every time a user tries to switch webpage. However, when the user is not logged in this is undesirable as they are unable to save the program anyway. Returning a rather annoying pop-up (every time!) a user switches Hedy levels. This is solved by adding an additional check to the `prompt_unsaved()` function to check if there is a `auth.profile` value. If so, we know that there is a user logged in, otherwise: don't return the pop-up.

**Fix for**

This PR fixed #1162

**How to test**

Create or alter a standard program in a Hedy level of choice when not being logged in. No pop-up should occur when switching Hedy levels. Verify the correctness by logging in and repeating the steps to find a pop-up showing up.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
